integration_test/cases/transaction_test.exs

```diff
@@ -965,10 +965,7 @@ defmodule TransactionTest do
 
     assert P.status(pool, opts) == :idle
     assert P.status(pool, opts) == :transaction
-    assert P.run(pool, fn(conn) ->
-      assert P.status(pool, [queue: false] ++ opts) == :error
-      assert P.status(conn, opts)
-    end, opts)
+    assert P.status(pool, [queue: false] ++ opts) == :error
```

Comment:
Thus doesn't test same thing, this test is that we get an `:error` atom when can't checkout from pool - so that handle_status is not called for this one.

Reply:
I moved this test below: https://github.com/elixir-ecto/db_connection/pull/145/files#diff-c000545f7e7485c4bc7ed2cc55e6f5b8R1021